src/components/Card/Card.styles.tsx

```
@@ -0,0 +1,41 @@
+    Box as MuiBox,
+    BoxProps as MuiBoxProps,
+    Card as MuiCard,
+    styled,
+} from '@mui/material';
+
+import { CardPaddingMap } from './Card.constants';
+import { CardProps } from './Card.types';
+
+export const StyledCard = styled(MuiCard)<CardProps>(
+    ({ theme: { palette, spacing }, size }) => ({
+        padding: spacing(CardPaddingMap[size!]),
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)